apps/site-v2/src/components/episode-container.tsx

```diff
@@ -87,6 +83,14 @@ export function EpisodeContainer({
           <source src={frontmatter.audioURL} type="audio/mpeg" />
           <track kind="captions" src={frontmatter.captionURL} />
         </audio>
+        <div className="flex flex-col md:flex-row gap-2 justify-center">
+          <Anchor href="https://open.spotify.com/show/7njrdM3LvNPnqSftswTkjn?si=5cc424416eaa4b35">
+            <Spotify className="w-4 h-4 mr-2" /> Listen on Spotify
+          </Anchor>
+          <Anchor href="https://podcasts.apple.com/us/podcast/the-bikeshed-pod/id1802688284">
+            <ApplePodcasts className="w-4 h-4 mr-2" /> Listen on Apple Podcasts
+          </Anchor>
```

Comment:
The less than ideal part of this change is that it only links to the show itself, not the episode directly 😢

Reply:
As far as I can tell - there's no deterministic way to generate the episode links for Spotify or Apple Podcasts